src/Security/Member.php

```diff
@@ -1246,7 +1246,7 @@ public static function mapInCMSGroups($groups = null)
             ]);
         }
 
-        return $members->sort('"Member"."Surname", "Member"."FirstName"')->map();
+        return $members->sort('"Surname", "FirstName"')->map();
```

Comment:
This is, ultimately, the fix this PR is for.

Reply:
Is this a legit non raw sql sort string? It doesn't match any of the examples in https://github.com/silverstripe/silverstripe-framework/blob/5/src/ORM/DataList.php#L324.  None of the examples use double quotes.

If it is legit, then add it to the list of examples in DataList.  Otherwise changes this to match one of the other examples

Reply:
It works, so I don't see how it wouldn't be "legit". If it shouldn't work it should explicitly _not_ work.
The `validateSortColumn` method does explicitly ignore double quotes. I'll add it to the examples I guess but I don't think we need to provide an exhaustive list there.

I'm not for example going to duplicate EVERY SINGLE EXAMPLE there so it shows with and without double quotes. I'll just dupe the first one.

Reply:
Were double quotes more a Postgres thing? If so, we could drop them here and not change the examples. But since this is already merged.. nvm.

Reply:
Yes, but the argument max made when we decided to drop official support for postgres was that we should still add double quotes so that a community-maintained postgres moduor would still work. 

I don't think that matters for this sort method (I suspect the lower levels of the ORM add quotes to it where needed)... but if it works now it's easier to let it keep working than to willfully break it.